python_examples/age_stratified_example/gibraltar_parameters.json

```diff
@@ -0,0 +1,407 @@
+{
+    "basic_reproduction_num": 2.8,
```

Comment:
Is this a direct duplication of the other Gibraltar parameters file? If so, just read in this other file as there's no need for a separate file. 

(In general if you consider this as a sub example of the Gibraltar simulation with age stratification, which I think it is, I think it would be best to restructure the directory (moving these workflows into a subfolder within that directory) to reflect that)

Reply:
Because my Gibraltar simulation was very much an adaption of Kits, it incorporates all and hence supersedes anything previous. I therefore have just one folder, "Gibraltar example" which should contain everything. The compatibility of the main flow file wil lallow us to turn things on/off as needed

Reply:
This is great - it sounds like we don't need to old version anymore then?

Reply:
Don;t think so... always tentative to delete though XD